stable/jenkins/README.md

```diff
@@ -73,12 +73,12 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `Master.JMXPort`                  | Open a port, for JMX stats           | Not set                                                                      |
 | `Master.ExtraPorts`               | Open extra ports, for other uses     | Not set                                                                      |
 | `Master.OverwriteConfig`          | Replace config w/ ConfigMap on boot  | `false`                                                                      |
-| `Master.HostName`                 | Enables chart ingress, ingress host path      | Not set                                                                         |
-| `Master.Ingress.ApiVersion`      | Ingress api version                  | Not set                                                                         |
-| `Master.Ingress.Annotations`      | Ingress annotations                  | `{}`                                                                         |
-| `Master.Ingress.Labels`           | Ingress labels                       | `{}`                                                                         |
-| `Master.Ingress.Path`             | Ingress path                         | Not set                                                                         |
-| `Master.Ingress.TLS`              | Ingress TLS configuration            | `[]`                                                                         |
+| `Master.ingress.Enabled`          | Enables ingress      | `false`                                                                         |
```

Comment:
enabled instead of Enabled

Reply:
fixed

Reply:
did you push these changes? i can't see them. same for the other 2 changes